app/routes/events/components/StripeElement.js

```diff
@@ -0,0 +1,102 @@
+//@flow
+import React from 'react';
+import {
+  injectStripe,
+  PaymentRequestButtonElement,
+  Elements,
+  StripeProvider
+} from 'react-stripe-elements';
+import config from 'app/config';
+import StripeCheckout from 'react-stripe-checkout';
+import Button from 'app/components/Button';
+import logoImage from 'app/assets/kule.png';
+
+type Props = {
+  event: Object,
+  currentUser: Object,
+  onToken: (token: string) => Promise<*>
+};
+
+type FormProps = Props & { stripe: { paymentRequest: Object => Object } };
+
+type State = {
+  paymentRequest: Object,
+  canMakePayment: boolean
+};
+
+class PaymentRequestForm extends React.Component<FormProps, State> {
+  constructor(props) {
+    super(props);
+
+    const { event, onToken } = props;
+
+    const paymentRequest = props.stripe.paymentRequest({
+      currency: 'nok',
+      total: {
+        label: event.title,
+        amount: event.price
+      },
+      requestPayerName: true,
+      requestPayerEmail: true,
+      requestPayerPhone: true,
+      country: 'NO'
+    });
+
+    paymentRequest.on('token', async ({ complete, token, ...data }) => {
+      await onToken(token);
+      complete('success');
+    });
+
+    paymentRequest.canMakePayment().then(result => {
+      this.setState({ canMakePayment: !!result });
+    });
+
+    this.state = {
+      canMakePayment: false,
+      paymentRequest
+    };
+  }
+
+  render() {
+    const { event, onToken, currentUser } = this.props;
+    return this.state.canMakePayment ? (
```

Comment:
If the payment request api isn't supported, it will fallback to the old `v2` version. Should move that to Stripe elements too...